app/src/pages/ExploreDataPage.tsx

```diff
@@ -49,6 +95,17 @@ function ExploreDataPage() {
         </Carousel>
       </div>
       <div className={styles.ReportContainer}>
+        <h1>
+          {getMadLibPhraseText(MADLIB_LIST[phraseIndex], phraseSelections)}
+          <IconButton
+            aria-label="delete"
+            color="primary"
+            onClick={() => setShareModalOpen(true)}
+            autoFocus
+          >
+            <ShareIcon />
```

Comment:
Is it intentional that the share icon is highlighted and lightly pulsing when you first load the page?

Reply:
good eye- copy paste artifact. fixed